src/richie/apps/search/filter_definitions/courses.py

```diff
@@ -368,8 +385,16 @@ def aggs_include(self):
         """
         if self.reverse_id:
             if self.base_page:
-                node = self.base_page.node
-                return f".*-{node.path:s}.{{{node.steplen:d}}}"
+                if not self._aggs_include:
```

Comment:
I thought `@cached_property` was not shared between requests but what is set on `self` was. Should we use django Local Memory Caching then? Maybe the same thing is appropriate for `base_page` too?

Reply:
I'm not sure actually it depends when the class is instantiated. The problem is that if it is cached then the problem is worse because we would have to restart all pods for a change to be taken into account.
So caching in local memory with a timeout seems better?

Reply:
Ok then, I'll use LocMemCache 👍